### PR TITLE
Add OVS/Kernel development workflow, including a simple network test

### PR DESCRIPTION
**Pull request by chaudron**

### CODE REVIEW

File: README_DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+```
+vagrant halt
+qemu-img resize ~/.local/share/libvirt/images/ovn-fake-<XXX>.img +100G
+vagrant up && vagrant ssh
+echo ", +" | sudo sfdisk -N 1 /dev/vda --no-reread
+sudo partprobe
+sudo sudo xfs_growfs /
+exit
+vagrant halt && vagrant up && vagrant ssh
```
Comment (flavio-fernandes) [inline README_DEVELOPMENT.md:34]: NIT: reload may apply here. ;)
```
vagrant reload && vagrant ssh
```

File: README_DEVELOPMENT.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
+# Using ovn-fake-multinode for development and testing
+
+## Setup the Vagrant environment
+
+This example explains how to use the ovn-fake-multinode environment when doing
+an Open vSwitch change, including a kernel module change. This assumes you are
+using the Vagrant environment.
```
Comment (flavio-fernandes) [inline README_DEVELOPMENT.md:7]: NICE! <3

File: README_DEVELOPMENT.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+```
+sudo ./setup_dev.sh -p -c -o
+```
+
+If the build is successful you can either quickly restart you vagrant VM,
+and try the traffic test:
+
+```
+exit
+vagrant halt && vagrant up && vagrant ssh
```
Comment (flavio-fernandes) [inline README_DEVELOPMENT.md:132]: NIT: vagrant reload suggestion, as above.

File: README_DEVELOPMENT.md (hunk 1 above)
Comment (chaudron) [reply README_DEVELOPMENT.md]: Fixed in next pash (3 locations).

File: README_DEVELOPMENT.md (hunk 3 above)
Comment (chaudron) [reply README_DEVELOPMENT.md]: Fixed in next pash (3 locations).

File: README_DEVELOPMENT.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+```
+vagrant halt
+qemu-img resize ~/.local/share/libvirt/images/ovn-fake-<XXX>.img +100G
+vagrant up && vagrant ssh
+echo ", +" | sudo sfdisk -N 1 /dev/vda --no-reread
+sudo partprobe
+sudo sudo xfs_growfs /
+exit
+vagrant reload && vagrant ssh && vagrant ssh
```
Comment (numansiddique) [inline README_DEVELOPMENT.md:34]: There is "vagrant ssh" twice. Is it intentional ?

File: README_DEVELOPMENT.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+...
+Last login: Mon Jul 13 14:00:18 2020 from 192.168.122.1
+[vagrant@ovnhostvm ~]$ uname -a
+Linux ovnhostvm 5.8.0-rc4+ #2 SMP Mon Jul 13 13:59:55 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
+```
+
+
+## Building Open vSwitch
+
+Open vSwitch is already built en deployed by the ovn_cluster.sh script. Here we
```
Comment (numansiddique) [inline README_DEVELOPMENT.md:117]: s/en/and

File: README_DEVELOPMENT.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+just explain what to do to re-build/re-deploy OVS is you made a change.
```
Comment (numansiddique) [inline README_DEVELOPMENT.md:118]: I'd rephrase this as
--
just explain what to do to re-build/re-deploy OVS after you made a change.
--

**Comment by chaudron** [comment]
Did another push, which clones and builds the kernel as the vagrant user rather than root, so you can edit, the code without switching privileges.

File: README_DEVELOPMENT.md (hunk 4 above)
Comment (chaudron) [reply README_DEVELOPMENT.md:34]: Nope, will fix in next commit

File: README_DEVELOPMENT.md (hunk 5 above)
Comment (chaudron) [reply README_DEVELOPMENT.md:117]: will fix

File: README_DEVELOPMENT.md (hunk 6 above)
Comment (chaudron) [reply README_DEVELOPMENT.md:118]: Thanks, will use your suggestion

**Comment by dceara** [comment]
@chaudron, thanks!
